tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -1201,4 +1201,42 @@ TEST_F(ChargepointTestFixtureV201, K01_ValidateAndAdd_AddsValidProfiles) {
     EXPECT_THAT(profiles, testing::Contains(profile));
 }
 
+TEST_F(ChargepointTestFixtureV201, K02FR05_SmartChargingTransactionEnds_DeletesTxProfilesByTransactionId) {
```

Comment:
The two tests in this file don't seem to be about a transaction ending, but rather the behavior of `delete_transaction_tx_profiles()`